rts/lib/lua/src/lvm.cpp

```diff
@@ -10,6 +10,7 @@
 #include <string.h>
 
 //SPRING
+#include <algorithm>
```

Comment:
for peace of mind, three more things should be done regarding this desync:

1) LC_NUMERIC should be changed to LC_ALL in https://github.com/spring/spring/blob/develop/rts/System/Main.cpp#L37
2) access to `os.setlocale` should be removed entirely (let https://github.com/spring/spring/blob/develop/rts/lib/lua/src/loslib.cpp#L231 `return 0`)
3) the locale should be net-logged somewhere on startup, for example at https://github.com/spring/spring/blob/develop/rts/Game/Game.cpp#L437 (change to `SNPRINTF(msgBuf, sizeof(msgBuf), "[Game::%s][lua{Rules,Gaia}={%p,%p}][locale=%s]", __func__, luaRules, luaGaia, setlocale(LC_ALL, nullptr));`)

Reply:
Sure, will do